src/components/ZoomButtons.tsx

```diff
@@ -43,6 +45,7 @@ const ZoomButton = ({
 		onMouseDown={start}
 		onMouseLeave={end}
 		onMouseUp={end}
+		title={restProps["title"]}
```

Comment:
Either:
* Just pick the title directly in the component props
* Spread the `...restProps`

If you're spreading the props, probably best to put that at the start of the component so that our required props aren't overridden.